src/components/List.js

```diff
@@ -74,6 +81,8 @@ const List = ({ results, setSearchTerm, searchTerm, token }) => {
                     className="checkbox"
                   />
                   {result.name}
+                  {/* this is just for me */}
```

Comment:
I think you may have left this in by accident!

Reply:
Yep, can remove.